code_examples/core_features/did/01_light_did_simple.ts

```diff
@@ -1,22 +1,24 @@
+import type { Keyring } from '@polkadot/api'
+
+import { randomAsU8a } from '@polkadot/util-crypto'
+
 import * as Kilt from '@kiltprotocol/sdk-js'
 
 export async function createSimpleLightDid(
-  keystore: Kilt.Did.DemoKeystore,
-  authenticationSeed: string | undefined = undefined
-): Promise<Kilt.Did.LightDidDetails> {
-  // Ask the keystore to generate a new keypair to use for authentication with the generated seed.
-  // If no `authenticationSeed` is provided, a random one will be generated.
-  const authenticationKeyPublicDetails = await keystore.generateKeypair({
-    alg: Kilt.Did.SigningAlgorithms.Ed25519,
-    seed: authenticationSeed
-  })
+  keyring: Keyring,
+  authenticationSeed: Uint8Array = randomAsU8a(32)
+): Promise<Kilt.DidDetails> {
+  // Ask the keyring to generate a new keypair to use for authentication with the generated seed.
+  const { publicKey } = keyring.addFromSeed(authenticationSeed, {}, 'ed25519')
```

Comment:
It is just a way to explicitely specify the type of a key, so that the same value can then be used when creating the light DID. If we use the default, we don't make very clear to the user what we are doing here. Do you suggest to still use `sr25519` as an explicit value?

Reply:
I think the used does not have to know this detail, so I would rather not make it explicit. Just let the code use the defaults.

Reply:
Yes but the DID key does not take a default, it has to take a `type` parameter, right? How is the user supposed to know that if the default is used?